src/structures/CiClient.ts

```diff
@@ -22,6 +23,16 @@ export class CiClient extends AkairoClient {
             directory: join(__dirname, "..", "events"),
         });
         this.commandHandler.useListenerHandler(this.eventHandler);
+        
+        this.commandHandler.resolver.addType('CiMembers', (message, phrase) => {
+            let members: GuildMember[] = []

```

Comment:
`member = phrase.split(' ').map(item => message.guild.members.cache.find(member => this.util.checkMember(item, member, false, true)))`